git/git.go

```diff
@@ -22,3 +23,37 @@ func Clone(repo, destination string) error {
 	}
 	return nil
 }
+
+// Fetch fetches a pull request into a branch named "_pr"
+func Fetch(prNumber string) error {
```

Comment:
something called Fetch shouldn't accept a pr number. Something called FetchPR could, but not something called Fetch